README.md

```diff
@@ -48,3 +40,7 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 * [Why do contract testing?](docs/why-contract-testing.md)
 * [Running your frontend against the examples and random content (content-store not needed)](docs/running-frontend-against-examples.md)
```

Comment:
Optional: this file still contains references to the dev VM. Do you think it's worth tidying it up as part of this PR?